test/josefsson-draft-vectors.config

```diff
@@ -0,0 +1,34 @@
+%% -*- mode: erlang; erlang-indent-level: 4; indent-tabs-mode: nil -*-
```

Comment:
I'm assuming this file is left untouched except for the occasional static analysis -related fix. Is my assumption correct?

Reply:
I don't understand your question - this is test data.